plugins/custom-class/index.html

```diff
@@ -61,6 +64,21 @@ <h2>2. Replace some Prism classes with your defined ones via an object</h2>
 	}
 });</code></pre>
 
+	<h2>Add new classes</h2>
+
+	<p>You can add new classes with per-token and per-language precision.</p>
+
+	<pre class="language-js"><code>Prism.plugins.customClass.add(function (content, type, language) {
```

Comment:
Looking at this API, I might suggest content go last, cuz its the least likely to be used by any particular implementation.

Reply:
To be clear, I think `type, language, content` is the preferred order.

Reply:
I actually changed the API from `language, type, content` to `content, type, language` in the last commit because I think it's the most useful one. Everything you can do with only `language` and `type`, you can also do in CSS (styling-wise), so content is the feature that enables you to do more.
Why do you think it's the least useful one?

> we could pass it in as an object with the three properties [from your other comment]

I like that. Makes it feel like a hook.